AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -11,39 +11,49 @@ import CoreData
 import Charts
 import AirCastingStyling
 
-struct SessionCellView: View {
+struct SessionCartView: View {
     
     @State private var isCollapsed = true
     @State private var selectedStream: MeasurementStreamEntity?
     
-    let session: SessionEntity
+    @ObservedObject var session: SessionEntity
     let thresholds: [SensorThreshold]
     
+    var shouldShowValues: Bool {
+        isCollapsed && (
+            (session.type == .fixed) ||                                  // Fixed
+                (session.type == .mobile && session.status == .FINISHED) // Dormant
+        ) ? false : true
```

Comment:
This is super hard to read and understand, lets use named helper variables to make it readable (compiler would optimize it anyway so it's generally a good idea to do so):

```Swift
let isExpanded = !isCollapsed
let isFixed = (session.type == .fixed)
let isDormant = (session.type == .mobile && session.status == .FINISHED)

return isExpanded || !(isFixed || isDormant)
```

Still not perfect, but at least somehow readable. Maybe it can be perfected further or even extracted to some kind of a `Decider` object?